src/tokens/functional/color/light/patterns-light.json5

```diff
@@ -1093,6 +1093,44 @@
           },
         },
       },
+      iconColor: {
+        rest: {
+          $value: '{fgColor.muted}',
+          $type: 'color',
+          $extensions: {
+            'org.primer.figma': {
+              collection: 'mode',
+              mode: 'light',
+              group: 'component',
+              scopes: ['fgColor'],
+            },
+          },
+        },
+        hover: {
+          $value: '{fgColor.muted}',
```

Comment:
In dark mode this is using `fgColor.default` shouldn't it be the same in both modes?

Reply:
Spoke offline- it looks pretty harsh in light mode and isn't required to pass contrast, so I'm only changing the color in dark mode. I think it feels more expected to see a lighter effect in a dark theme on hover vs. darker in light mode.